var/spack/repos/builtin/packages/swan/package.py

```diff
@@ -0,0 +1,37 @@
+from spack import *
+
+class Swan(MakefilePackage):
```

Comment:
You might want to consider leveraging the `SourceforgePackage` mix-in.  See https://spack.readthedocs.io/en/latest/_modules/spack/build_systems/sourceforge.html?highlight=SourceforgePackage.

There are currently 44 package that use it.  Examples include `octave-*` packages, `swig`, `tcl`, and `tk`.

Reply:
Thanks for you review. But I don't know much about `class SourceForge`. So I think it's better to keep it as `class Swan(MakefilePackage):`